book/02-git-basics/sections/tagging.asc

```diff
@@ -217,6 +217,25 @@ To git@github.com:schacon/simplegit.git
 
 Now, when someone else clones or pulls from your repository, they will get all your tags as well.
 
+==== Deleting Tags
+
+To delete a tag on your local repository, you can use `git tag -d <tagname>`. For example, we could remove our lightweight tag above as follows: 
```

Comment:
This is _almost_ perfect. Would you reformat so that each sentence gets its own source line? It makes diffing in the future much friendlier.